src/routes/docs/tooling/command-line/installation/+page.markdoc

```diff
@@ -70,6 +70,29 @@ Before you can use the CLI, you need to login to your Appwrite account using
 appwrite login
 ```
 
+In case of a self-hosted instance, add the `self-hosted` flag.
+
+```sh
+appwrite login --self-hosted
+```
```

Comment:
Where and how is this `self-hosted` flag used?

Reply:
This is now `--endpoint="[YOUR_ENDPONT_HERE]"`